_data/api-tools.yml

```diff
@@ -43,6 +43,24 @@
   runtime: "https://img.shields.io/badge/API_Runtime-No-red.svg"
   testing: "https://img.shields.io/badge/API_Testing-Yes-brightgreen.svg"
   notes: ""
+- name: "API Protection on AppTrana"
+  link: "https://www.indusface.com/api-protection.php"
+  from: "Indusface"
+  license: "https://img.shields.io/badge/license-Commercial-9cf.svg"
+  platforms: "https://img.shields.io/badge/Platform-SaaS%2C%20Cloud-inactive.svg"
+  posture: "https://img.shields.io/badge/API_Posture-Yes-brightgreen.svg"
+  runtime: "https://img.shields.io/badge/API_Runtime-Yes-brightgreen.svg"
+  testing: "https://img.shields.io/badge/API_Testing-Yes-brightgreen.svg"
+  notes: "<br/>AI-powered, fully managed API protection against OWASP API Top 10 threats, DDoS and bot attacks. Unified platform for API discovery, vulnerability scanning and protection"
+- name: "API Scanning"
+  link: "https://www.indusface.com/api-scanning.php"
+  from: "Indusface"
+  license: "https://img.shields.io/badge/license-Commercial-9cf.svg"
+  platforms: "https://img.shields.io/badge/Platform-SaaS%2C%20Cloud-inactive.svg"
+  posture: "https://img.shields.io/badge/API_Posture-Yes-brightgreen.svg"
+  runtime: "https://img.shields.io/badge/API_Runtime-Yes-brightgreen.svg"
+  testing: "https://img.shields.io/badge/API_Testing-Yes-brightgreen.svg"
+  notes: "<br/>A complete vulnerability analysis and penetration testing platform for APIs"
```

Comment:
I don't think it make sense for posture/runtime/testing to all be brightgreen for both products. If they both fulfill all three goals why have two products?

> Types of API Tools
> Tools for API Security can be broken down into 3 broad categories.
> 
> API Security Posture: Creates an inventory of APIs, the methods exposed and classifies the data used by each method.
> Goal: Provide visibility into the security state of a collection of APIs.
> API Runtime Security: provides protection to APIs during their normal running and handling of API requests.
> Goal: Detect and prevent malicious requests to an API.
> API Security Testing: Dynamic assessment of an API’s security state.
> Goal: Evaluate the security of a running API by interacting with the API dynamically (DAST-like behavior)

Reply:
Agreed. The two entries should be collapsed into a single entry or each entry should only show what it covers in those 3 broad categories.

Reply:
@kingthorin did changes here please check and approve it https://github.com/OWASP/www-community/pull/1016/commits/d06a9fbb951bd17e87e48c1884880901c332a685